### ISSUE TITLE
The secureTransport.sendRequest can't pass although the secureDataDescriptors is correct

### ISSUE THREAD

**Issue by haoxli** [01org/AppSecurityApi#4]
sendRequest() goes to error callback when set secureDataDescriptors to:

```
[{'Path': 'password', 'Type': 'String', 'Handle': passwordSecureDataInstanceID, 'Format': 'SECURE_DATA_PLAINTEXT_UNDECORATED'}]
```

It will throw error message:

fail: code=19, message= Invalid descriptor structure

Test Code:

```
var request = {'username': 'John Smith', 'password': ''};
var requestDescriptors = [{'Path': 'password', 'Type': 'String', 'Handle': passwordSecureDataInstanceID, 'Format': 'SECURE_DATA_PLAINTEXT_UNDECORATED'}];
intel.security.secureTransport.sendRequest(
    function(response){
        t.step(function() {
            assert_not_equals(response.responseBody, null);
        });
        t.done();
    },
    function(errorObj){
        t.step(function() {
            assert_true(false, 'fail: code = ' + errorObj.code + ', message = ' + errorObj.message);
        });
        t.done();
    },
    {'instanceID': secureTransportInstanceID, 'requestBody': JSON.stringify(request), 'requestFormat': 'JSON', 'secureDataDescriptors': requestDescriptors}
);
```

**Comment by OhadBoimelIntel** [comment]
Hi @haoxli,
What plugin version are you using? is it 1.3.0? 

Thank, 
Ohad

**Comment by haoxli** [comment]
yes, the latest 1.3.0

**Comment by OhadBoimelIntel** [comment]
We acknowledge the issue, it will be fixed in future version

**Comment by OhadBoimelIntel** [comment]
Fixed in version 1.4.0
